[2022-04-13 22:36:24,477] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: LocalIngestionDAG.ingest scheduled__2022-04-02T06:00:39+00:00 [queued]>
[2022-04-13 22:36:24,535] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: LocalIngestionDAG.ingest scheduled__2022-04-02T06:00:39+00:00 [queued]>
[2022-04-13 22:36:24,539] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-04-13 22:36:24,541] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-04-13 22:36:24,543] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-04-13 22:36:24,629] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): ingest> on 2022-04-02 06:00:39+00:00
[2022-04-13 22:36:24,645] {standard_task_runner.py:52} INFO - Started process 11406 to run task
[2022-04-13 22:36:24,660] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'LocalIngestionDAG', 'ingest', 'scheduled__2022-04-02T06:00:39+00:00', '--job-id', '1972', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_local.py', '--cfg-path', '/tmp/tmpjt4mnv68', '--error-file', '/tmp/tmpc8vt2ddl']
[2022-04-13 22:36:24,667] {standard_task_runner.py:77} INFO - Job 1972: Subtask ingest
[2022-04-13 22:36:24,939] {logging_mixin.py:109} INFO - Running <TaskInstance: LocalIngestionDAG.ingest scheduled__2022-04-02T06:00:39+00:00 [running]> on host 13db72185839
[2022-04-13 22:36:25,148] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=LocalIngestionDAG
AIRFLOW_CTX_TASK_ID=ingest
AIRFLOW_CTX_EXECUTION_DATE=2022-04-02T06:00:39+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-04-02T06:00:39+00:00
[2022-04-13 22:36:25,151] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-04-13 22:36:25,153] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'ls /opt/***']
[2022-04-13 22:36:25,176] {subprocess.py:85} INFO - Output:
[2022-04-13 22:36:25,183] {subprocess.py:89} INFO - ***.cfg
[2022-04-13 22:36:25,184] {subprocess.py:89} INFO - dags
[2022-04-13 22:36:25,185] {subprocess.py:89} INFO - logs
[2022-04-13 22:36:25,185] {subprocess.py:89} INFO - output.csv
[2022-04-13 22:36:25,186] {subprocess.py:89} INFO - output_2022-03.csv
[2022-04-13 22:36:25,187] {subprocess.py:89} INFO - output_2022-04.csv
[2022-04-13 22:36:25,188] {subprocess.py:89} INFO - plugins
[2022-04-13 22:36:25,189] {subprocess.py:89} INFO - requirements.txt
[2022-04-13 22:36:25,190] {subprocess.py:89} INFO - scripts
[2022-04-13 22:36:25,192] {subprocess.py:89} INFO - webserver_config.py
[2022-04-13 22:36:25,194] {subprocess.py:93} INFO - Command exited with return code 0
[2022-04-13 22:36:25,552] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=LocalIngestionDAG, task_id=ingest, execution_date=20220402T060039, start_date=20220413T223624, end_date=20220413T223625
[2022-04-13 22:36:25,650] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-13 22:36:25,777] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
